sig-testing/README.md

```diff
@@ -26,6 +26,13 @@ Interested in how we can most effectively test Kubernetes. We're interested spec
 * [Mailing list](https://groups.google.com/forum/#!forum/kubernetes-sig-testing)
 * [Open Community Issues/PRs](https://github.com/kubernetes/community/labels/sig%2Ftesting)
 
+## Subprojects
+
+The following subprojects are owned by this SIG:
+- **test-infra**
```

Comment:
LGTM

Reply:
What about test infra in the kubernetes repo, such as parts of https://github.com/kubernetes/kubernetes/tree/master/test/e2e?

Reply:
For now I have put all of k/k/test under the "testing-commons" subproject, rather than sharding by subfolder